packaging/nfpm/verify.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+# Copyright 2021 Tetrate
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euo pipefail
+
+# The packaging type. This can be .rpm or .deb.
+PACKAGING=${PACKAGING:-deb}
```

Comment:
I think this file is over-normalized. To figure out what's going on requires thinking too much, like which variables are passing to the almost identical two workflows and do they or do they not result in a rpm or deb run?

Let's keep it more straightforward.. how about verify_rpm verify_deb instead? the cost of duplicating `func-e -version || exit 1` a couple more times is worth buying the straight-forwardness